accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/configreader/YamlMacroProcessorImpl.java

```diff
@@ -238,7 +229,12 @@ private Object evaluateIfStatement(Map<String, Object> variables, Object objVal,
             InstallationLogger installLog, Session session) {
         String condition = ifMatcher.group(1).trim();
 
-        boolean expressionIsTrue = elEvaluator.evaluateEl(condition, Boolean.class, variables);
+        Boolean expressionIsTrue = elEvaluator.evaluateEl(condition, Boolean.class, variables);
+
+        if (expressionIsTrue == null) {
+            LOG.warn("Expression {} evaluates to null, returning false", condition);

```

Comment:
we should use the methods in installLog for logging (they log both to the error.log and to the AC Tool log available in the system console)